scripts/fiber/tests-failing.txt

```diff
@@ -9,6 +9,10 @@ src/addons/__tests__/ReactFragment-test.js
 src/isomorphic/classic/__tests__/ReactContextValidator-test.js
 * should pass previous context to lifecycles
 
+src/isomorphic/modern/class/__tests__/ReactClassEquivalence-test.js
+* tests the same thing for es6 classes and CoffeeScript
+* tests the same thing for es6 classes and TypeScript
+
```

Comment:
No, the earlier commit wouldn't have made it past CI (unless someone merged without noticing). I just ran master to verify and the fiber snapshots look right.

I also just ran your branch locally and the fiber script moved those 2 entries _out_ of the failing tests and back into the passing ones. Did you perhaps run it with some other local/uncommitted changes?